pathspider/plugins/ecnspider3.py

```diff
@@ -155,6 +169,47 @@ def create_observer(self):
             traceback.print_exc()
             sys.exit(-1)
 
+    def combine_flows(self, flow):
```

Comment:
Possibly. Not yet, though: I don't want the base Spider to have to make assumptions about the ports/addresses linkage just yet (especially as we'll need to change how linkage works to fix #43). Seems like a good candidate for later refactoring though.